### ISSUE TITLE
Zammad setup guide supports any email channel

### ISSUE THREAD

**Issue by mantas** [zammad/zammad#4073]
### Infos:

* Used Zammad version: 5.2
* Installation method (source, package, ..): any
* Operating system: any
* Database + version: any
* Elasticsearch version: any
* Browser + version: any

### Expected behavior:

* Zammad can be set up with any supported email channel

### Actual behavior:

* Zammad requires using Sendmail or SMTP during setup
* Setup guide does not allow to skip this step

![Screenshot 2022-04-29 at 17 32 04](https://user-images.githubusercontent.com/36944/165965822-efd143c0-0378-48a3-9918-4916181a7ece.png)

### Steps to reproduce the behavior:

* Start Zammad setup from scratch
* Get to Email Notification screen
* Evaluate email sending options

Yes I'm sure this is a bug and no feature request or a general question.

**Comment by MrGeneration** [comment]
Sorry I don't understand the issue scope.
Supported channels for notifications are: `sendmail` and `smtp`. These are available in the getting started wizard.

Documentation also covers this parts and suggests to skip in case something would not be supported by the email infrastructure:
https://docs.zammad.org/en/latest/getting-started/first-steps.html


This smells a lot like something PM should discuss.

**Comment by mantas** [comment]
Yes, this is supposed to be a placeholder for something PM shall discuss. @dominikklein is this a wrong place for this?

UX may need improvements since I couldn't find a way to skip it. I had a (legit) error with email gateway and was stuck.

It feels confusing if I have Gmail/Office 365 credentials and try to set up Zammad. Few words could help to understand what's the difference between Email Notifications and Email Channels.

Possibility to send Email notifications over MS365 or Google would be an interesting topic as well. Given that XAuth is slowly becoming mandatory.

**Comment by martini** [comment]
It is about "email channels" or about "notification channel"? Maybe a full description with maybe screenshots makes more sens in public issues (am,.. no feature enhancements).

**Comment by MrGeneration** [comment]
Gmail and Microsoft XOAUTH were not part of the implementation notification channel wise.
This is a feature request. We've decided against implementing it in the past, as Microsoft and Google are using sendout limits.

This can be very troublesome with notification channels as Zammad is very very communicative.
The documentation also states this fact (as per why this isn't possible as of now) ...

**Comment by arakelian** [comment]
Can I ask why is there an "Outbound" section on the Microsoft 365 account configuration screen, if it is not used for email notifications?  Are Microsoft 365 email accounts' outbound settings ever used, if not for notification?

**Comment by MrGeneration** [comment]
@arakelian this repository is not the right place for technical questions. Please consult the community with your technical questions: https://community.zammad.org

**Comment by dominikklein** [comment]
Is this not a duplicate of https://github.com/zammad/zammad/issues/3236?

**Comment by MrGeneration** [comment]
I'd say so too.
